certified-connectors/OzyApprovals/apiDefinition.swagger.json

```diff
@@ -1 +1,1285 @@
-
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Ozy Approvals",
+    "description": "Manage approvals like a pro.",
+    "version": "1.0",
+	"contact":{
+		"name": "OzyApprovals Support",
+		"url": "guide.ozyapprovals.com",
+		"email" : "support@ozyapprovals.com"
+	}
+  },
+   "x-ms-connector-metadata":[
+	  {
+		  "propertyName":"Website",
+		  "propertyValue":"https://ozyapprovals.com"
+	  },
+	  {
+		  "propertyName":"Privacy policy",
+		  "propertyValue":"https://ozyapprovals.com/index.php/home/privacy_policy"
+	  },
+	  {
+		  "propertyName":"Categories",
```

Comment:
The connector category is invalid. Available categories are: [AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website]